pkgs/development/tools/purescript/spago/spago-0.12.1-dhall-1.28.0.patch

```diff
@@ -0,0 +1,122 @@
+--- a/src/Spago/Config.hs
```

Comment:
If possible, we generally don't like to carry around patches in the nixpkgs repo, but instead fetch them from upstream.

Could you pull this patch directly from upstream and apply it using `fetchPatch`.  If you grep through `pkgs/development/haskell-modules/configuration-common.nix` you should be able to find examples of people pulling in patches from GitHub PRs.

Reply:
spacchetti/spago#529 cannot be applied for spago-0.12.1. different patch required.

Reply:
@ijaketak Please comment that on top of the patch.